src/AutoRest.CSharp/Common/Generation/Writers/CodeWriterExtensions.cs

```diff
@@ -354,38 +392,25 @@ void WriteCollectionInitializer(CodeWriter writer1, CodeWriterDeclaration codeWr
             return writer;
         }
 
-        public static void WriteConversion(this CodeWriter writer, CSharpType from, CSharpType to, CodeWriterDelegate value)
+        public static CodeWriter WriteConversion(this CodeWriter writer, CSharpType from, CSharpType to)
```

Comment:
I had it as a callback in case we ever need to wrap the value. For example `new Something({value})`

Reply:
We can always add an overload later if we need it. So far, it isn't used but makes code that uses method harder to read.